app/share.js

```diff
@@ -78,18 +79,37 @@ class Root extends React.Component {
 
 	render() {
 		const { isLandscape } = this.state;
+		if (isAndroid) {
```

Comment:
We really need this condition? I think we can use directly `<LayoutAnimation>`

Reply:
🤦‍♂ Fixed